.cirrus.yml

```diff
@@ -226,11 +226,28 @@ publish_images_task:
             sha256sum * > shasums
             popd
 
+            if [[ $CIRRUS_TAG == *-rc* ]]; then
+                RC="--prerelease"
+            else
+                # check if this version should not be marked latest
+                prevrelease=$(curl --retry 3 --silent -m 10 --connect-timeout 5 "https://api.github.com/repos/containers/podman-machine-os/releases/latest")
+                prevvers=$(echo "$prevrelease" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/' | sed -e "s/^v//")
+                vers=${CIRRUS_TAG#"v"}
+                echo "${prevvers},${vers}"
+                # sort -V -C returns 0 if args are ascending version order
+                if !(echo "${prevvers},${vers}" | tr ',' '\n' | sort -V -C)
```

Comment:
Also I don't think you must trim the leading `v` for the sort because all tags have the leading v they should be all consider equal on the first char?